docs/projects/lottie/layers/radial_gradient.rst

```diff
@@ -0,0 +1,22 @@
+Radial Gradient Layer
+=====================
+
++---------------------+---------------------------------------------------------------------------+
+|       Property      |                         Path to Property in lottie                        |
++=====================+===========================================================================+
+|       Z depth       |               Depends on ordering of layers in lottie format              |
++---------------------+---------------------------------------------------------------------------+
+|        Amount       |   layers/shape.json -> "ef" -> effects/gFill.json -> effects/opacity.json |
++---------------------+---------------------------------------------------------------------------+
+|     Blend_method    |Not supported                                                              |
++---------------------+---------------------------------------------------------------------------+
+|        Gradient     |      layers/shape.json -> "ef" -> effects/gfill.json                      |
++---------------------+---------------------------------------------------------------------------+
+|       Center        |      layers/shape.json ->  "ef" -> effects/gFill.json ->  "s"             |
```

Comment:
Show how you used center and radius as you have mentioned in the comments in the code: https://github.com/synfig/synfig/pull/1446/files#diff-5a70b0c8e040579359b43c0cfd55b9f9R69

Reply:
I have added the description for the calculation of gradient start point and gradient end point in the important points section.